pronto_mia/lib/core/services/deployment_plan_service.dart

```diff
@@ -49,7 +49,7 @@ class DeploymentPlanService with ChangeNotifier {
 
   Future<List<DeploymentPlan>> getAvailableDeploymentPlans() async {
     final queryVariables = {
-      'availableUntil': DateTime.now().toIso8601String(),
+      'availableUntil': DateTime.now().toUtc().toIso8601String(),

```

Comment:
Please add a ToDo-Comment containing the HotChocolate issue so that we might get back to it as soon as it is resolved.

Also have you tested, that the resulting string contains `+00:00`?

Reply:
ToDo comment was added for this issue and also for the graphql issue.

Iso8601String doesn't contain any + timezone at all:
![image](https://user-images.githubusercontent.com/12544936/118299936-b0b18f80-b4e1-11eb-90a7-5d55ac8749e5.png)

Reply:
It does, the Z stands for `Zulu` and is the timezone. But this should be good 👍🏽